manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -288,6 +308,12 @@ spec:
           name: {{ .configMapName | quote }}
           optional: true
       {{- end }}
+      {{- if ne .Values.global.keyfactor.secretName ""}}
```

Comment:
Same - we don't want the template to have one section for each plugin. 

There should be a way to specify a list of secrets to be mounted - independent of any vendor.